client/cmd/dexc-desktop/main.go

```diff
@@ -0,0 +1,556 @@
+	m := windowManager
+
+	m.Lock()
+	m.counter++
+	windowID := windowManager.counter
+	m.windows[windowID] = cmd
+	windowCount := len(m.windows)
+	m.Unlock()
+	defer closeWindow(windowID)
+	log.Info("Opening new window. %d windows open now", windowCount)
+	cmd.Run()
+}
+
+func findExePath() string {
+	rawPath, err := os.Executable()
+	if err != nil {
+		panic("error finding executable: " + err.Error())
+	}
+	s, err := filepath.EvalSymlinks(rawPath)
+	if err != nil {
+		panic("error resolving symlinks:" + err.Error())
+	}
+	return s
+}
+
+//go:embed src/dexc.png
+var FavIcon []byte
+
+//go:embed src/symbol-bw-round.png
+var SymbolBWIcon []byte
+
+func systrayOnReady(ctx context.Context, logDirectory string, openC chan<- struct{}, killC chan<- os.Signal) {
+	go func() {
+		<-ctx.Done()
+		systray.SetTooltip("Shutting down. Please wait...")
+		systray.Quit()
+	}()
+
+	systray.SetIcon(FavIcon)
+	systray.SetTitle("DEX client")
+	systray.SetTooltip("Self-custodial multi-wallet")
+
+	// TODO: Consider reworking main so we can show the icon earlier?
+	// mStarting := systray.AddMenuItem("Starting...", "Starting up. Please wait...")
+	// var addr string
+	// var ok bool
+	// select {
+	// case addr, ok = <-webserverReady:
+	// 	if !ok { // no webserver started
+	// 		fmt.Fprintln(os.Stderr, "Web server required!")
+	// 		cancel()
+	// 		return
+	// 	}
+	// case <-mainDone:
+	// 	return
+	// }
+
+	// mStarting.Hide()
+
+	mOpen := systray.AddMenuItem("Open", "Open DEX client window")
+	mOpen.SetIcon(SymbolBWIcon)
+	go func() {
+		for range mOpen.ClickedCh {
+			select {
+			case openC <- struct{}{}:
+				log.Debug("Received window reopen request from system tray")
+			default:
+				log.Infof("Ignored a window open request from the system tray")
+			}
+		}
+	}()
+
+	systray.AddSeparator()
+
+	if logDirURL, err := app.FilePathToURL(logDirectory); err != nil {
+		log.Errorf("error constructing log directory URL: %v", err)
+	} else {
+		mLogs := systray.AddMenuItem("Open logs folder", "Open the folder with your DEX logs.")
+		go func() {
+			for range mLogs.ClickedCh {
+				runWebviewSubprocess(ctx, logDirURL)
+			}
+		}()
+	}
+
+	// TODO: Allow editing of configuration? What happens when they screw it up
+	// and make startup impossible?
+	// if cfgPathURL, err := app.FilePathToURL(cfgPath); err != nil {
+	// 	fmt.Fprintln(os.Stderr, err)
+	// } else {
+	// 	mConfigFile := systray.AddMenuItem("Edit config file", "Open the config file in a text editor.")
+	// 	go func() {
+	// 		for range mConfigFile.ClickedCh {
+	// 			if _, err := os.Stat(cfgPath); err != nil {
+	// 				if os.IsNotExist(err) {
+	// 					fid, err := os.Create(cfgPath)
+	// 					if err != nil {
+	// 						fmt.Fprintf(os.Stderr, "failed to create new config file: %v", err)
+	// 						continue
+	// 					}
+	// 					fid.Close()
+	// 				}
+	// 			}
+	// 			err := browser.OpenURL(cfgPathURL)
+	// 			if err != nil {
+	// 				fmt.Fprintln(os.Stderr, err)
+	// 			}
+	// 		}
+	// 	}()
+	// }
```

Comment:
Just rm these lines then.  They are there for reference in main_tray.go if we change our mind.  But my answer to what if they screw it up is to have a link to the config file in the OS launcher, or just have a warning in the config file when they edit it.  Besides, if they edited it to screw it up, they know where it is.